### ISSUE TITLE
Uncaught exception: 'list' object has no attribute 'decode'

### ISSUE THREAD

**Issue by HerHde** [nicfit/eyeD3#335]
Hey there,
I've just installed the `python-eyed3  1:0.8-4` package on my Manajaro Linux and encountered a problem using `eyeD3 *.mp3`.
```
$ python --version
Python 3.7.3

$ eyeD3 --version
0.8
```

```
eyed3.core:WARNING: Invalid UFID, owner_id is empty. Setting to 'http://www.id3.org/dummy/ufid.html'
[redacted]/01.mp3           [ 4.54 MB ]
-------------------------------------------------------------------------------
Time: 03:12	MPEG1, Layer III	[ ~196 kb/s @ 44100 Hz - Joint stereo ]
-------------------------------------------------------------------------------
ID3 v2.3:
title:  new andromeda theory
artist: wasaru
album: undefinable scenes (2008)
album artist: None
original release date: 2008-12-31
recording date: 2008
encoding date: 2009-01-18
track: 1/9		genre: Instrumental (id 33)
Uncaught exception: 'list' object has no attribute 'decode'

eyed3:ERROR: 'list' object has no attribute 'decode'
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/eyed3/main.py", line 279, in _main
    retval = mainFunc(args, config)
  File "/usr/lib/python3.7/site-packages/eyed3/main.py", line 50, in main
    fs_encoding=args.fs_encoding)
  File "/usr/lib/python3.7/site-packages/eyed3/utils/__init__.py", line 81, in walk
    handler.handleFile(os.path.abspath(path))
  File "/usr/lib/python3.7/site-packages/eyed3/plugins/classic.py", line 486, in handleFile
    self.printTag(self.audio_file.tag)
  File "/usr/lib/python3.7/site-packages/eyed3/plugins/classic.py", line 631, in printTag
    ufid.uniq_id.decode("unicode_escape")))
AttributeError: 'list' object has no attribute 'decode'
```

Hope this is of use for you!

**Comment by nicfit** [comment]
Thanks for reporting @HerHde , if you get the chance try parsing this file with a newer version of eyeD3, 0.8 is pretty dated at this point.

**Comment by HerHde** [comment]
Sorry, I've just seen that it's already marked outdated in the [Arch Packages](https://www.archlinux.org/packages/community/any/python-eyed3/). **However**, version `0.8.10` from pip (`pip install --user eyed3`) doesn't work either:

```
$ /home/herhde/.local/bin/eyeD3 --remove-all-images /tmp/01.mp3 
eyed3.plugins:WARNING: Plugin '('lastfm.py', '/home/herhde/.local/lib/python3.7/site-packages/eyed3/plugins')' requires packages that are not installed: cannot import name 'COVER_EXTRA_LARGE' from 'pylast' (/usr/lib/python3.7/site-packages/pylast/__init__.py)
eyed3.core:WARNING: Invalid UFID, owner_id is empty. Setting to 'b'http://www.id3.org/dummy/ufid.html''
/tmp/01.mp3                                                                                                                          [ 4.54 MB ]
-------------------------------------------------------------------------------------------------------------------------------------------------
Removing all images...
Time: 03:12	MPEG1, Layer III	[ ~196 kb/s @ 44100 Hz - Joint stereo ]
-------------------------------------------------------------------------------------------------------------------------------------------------
ID3 v2.3:
title:  new andromeda theory
artist: wasaru
album: undefinable scenes (2008)
original release date: 2008-12-31
recording date: 2008
encoding date: 2009-01-18
track: 1/9		genre: Instrumental (id 33)
Uncaught exception: 'list' object has no attribute 'decode'

eyed3:ERROR: 'list' object has no attribute 'decode'
Traceback (most recent call last):
  File "/home/herhde/.local/lib/python3.7/site-packages/eyed3/main.py", line 277, in _main
    retval = mainFunc(args, config)
  File "/home/herhde/.local/lib/python3.7/site-packages/eyed3/main.py", line 49, in main
    fs_encoding=args.fs_encoding)
  File "/home/herhde/.local/lib/python3.7/site-packages/eyed3/utils/__init__.py", line 108, in walk
    handler.handleFile(os.path.abspath(path))
  File "/home/herhde/.local/lib/python3.7/site-packages/eyed3/plugins/classic.py", line 493, in handleFile
    self.printTag(self.audio_file.tag)
  File "/home/herhde/.local/lib/python3.7/site-packages/eyed3/plugins/classic.py", line 655, in printTag
    ufid.uniq_id.decode("unicode_escape")))
AttributeError: 'list' object has no attribute 'decode'

```

The tested file is https://archive.org/download/jamendo-037609/01.mp3 (CC BY-NC-SA Wasaru)

**Comment by nicfit** [comment]
Thanks for the test file. Actually this bug as has been fixed prior in #325. I was not planning on backporting to 0.8, but would so with a PR. Otherwise, perhaps `pip install eyeD3==0.9a1` works for you. This version does not support Python2.
